habitat-lab/habitat/articulated_agents/humanoids/kinematic_humanoid.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List
+
+import magnum as mn
+import numpy as np
+
+import habitat_sim.physics as phy
+from habitat.articulated_agents.mobile_manipulator import (
+    ArticulatedAgentCameraParams,
+    MobileManipulator,
+    MobileManipulatorParams,
+)
+
+
+class KinematicHumanoid(MobileManipulator):
+    def _get_humanoid_params(self):
+        return MobileManipulatorParams(
+            arm_joints=[],  # For now we do not add arm_joints
+            gripper_joints=[],
+            wheel_joints=None,
```

Comment:
Thank you for working on this! Do we expect that humans have wheels?

Reply:
No, but I found I could not ignore this parameter, even if it was optional. I think that in order to be able to ignore it, we need to set a default argument to wheels, [here](https://github.com/facebookresearch/habitat-lab/blob/main/habitat-lab/habitat/articulated_agents/mobile_manipulator.py#L76) of None. I can do that.

Reply:
We have [here](https://github.com/facebookresearch/habitat-lab/blob/0cf9e5fae5873d18ed6fde2f93c9694a3bd0f667/habitat-lab/habitat/articulated_agents/articulated_agent_base.py#L83) a check of wheel_joints existing as a param or being `None`, this is why I was proposing to set the default to `None`. If we set it as `False`, or `[]`, we would need to change all these checks as well. I can do that if you prefer.

Reply:
What is the type of this variable? This should indicate what the no wheel option should be.

Reply:
`Optional[List[int]]`

Reply:
yeah, I faced the same issue before, my solution at that time was to use inheritance, which is not the best way to do this

Reply:
Could you elaborate on how you used inheritance? I think the best would be to make it List[int] (no optional) with a default argument `[]` and whenever we are checking this param, we replace the `None/False` checks fro empty list.